src/vhip/mlmodel/compute_ml_features.py

```diff
@@ -300,6 +301,42 @@ def generate_kmer_profiles(self):
             seq_profile.generate_profile()
             self.k6profiles[host] = seq_profile
 
+    def generate_codon_frq(
+        self, threshold_imprecise: float = 0.0, threshold_skipped_genes: float = 0.5
+    ) -> None:
+        """Generate profile of the frequency of each unique codon in every virus and host GeneSet.
+
+        This will be done for each .ffn files in the virus and host genes files directories.
```

Comment:
Can you clarify this? Did you mean for each genes file in the .ffn file?

Reply:
Changed. Does that make more sense?